LunaDll/Globals.h

```diff
@@ -205,3 +205,5 @@ return;
 
 
 #endif
+
+extern std::string editorPlacedItem;
```

Comment:
When making new global variables please use the `g` prefix like the other global variables generally have.